karma.conf.js

```diff
@@ -1,33 +1,39 @@
-module.exports = function (config) {
-    config.set({
-        browsers: ['PhantomJS'],
+module.exports = (config) => {
```

Comment:
Branches are based off other branches, the node version got updated in the next chain which allows things to work.